code/modules/events/holiday/vday.dm

```diff
@@ -60,15 +60,27 @@
 	               "If I were the warden I'd always let you into my armory.",
 	               "The virologist is rogue, and the only cure is a kiss from you.",
 	               "Would you spend some time in my upgraded sleeper?",
-	               "You must be a silicon, because you've unbolted my heart."
-	               "Are you Nar-Sie? Because there's nar-one else I sie." )
+	               "You must be a silicon, because you've unbolted my heart.",
+	               "Are you Nar-Sie? Because there's nar-one else I sie.",
+	               "If you were a taser, you'd be set to stunning.",
```

Comment:
i'm laughing a fucking lot holy shit i will use this and get immediately slapped